hw_8/mkdir.py

```diff
@@ -0,0 +1,31 @@
+#! /root/miniconda3/bin/python
+
+import sys
+import os
+import shutil
+import argparse
+
+# create arguments - directory and flag for parent directories
+
+parser = argparse.ArgumentParser(
+                    prog='mkdir analog',
+                    description='Program create directory',
+                    epilog='Text at the bottom of help')
+
+parser.add_argument('-p', '--parents', action='store_true')
+parser.add_argument('directories', nargs='+')
+
+args = parser.parse_args()
+
+for directories in args.directories:
+    if args.parents:                                                       # for directory with intermediate directories
+        creation = os.path.join(os.getcwd(), directories)
+        os.makedirs(creation)
```

Comment:
При попытке создания уже существующей папки с флагом `-p` происходит ошибка, которая к тому же ещё и выкидывается юзеру. Линуховый `mkdir` такие ситуации не смущают 
![image](https://user-images.githubusercontent.com/48160953/209410574-137d5fa0-e056-4078-ab21-46885872a16f.png)

```suggestion
        os.makedirs(creation, exist_ok=True)
```